test/mod/test_util_chroot.py

```diff
@@ -0,0 +1,34 @@
+#
+# Test for util/chroot.py
+#
+
+from unittest.mock import patch
+
+from osbuild.util.chroot import Chroot
+
+
+class RunReturn:
+    """
+    Class to be returned from mocked run() call so that the returncode is always 0.
+    """
+
+    @property
+    def returncode(self):
+        return 0
+
+
+@patch("subprocess.check_call")
+@patch("subprocess.run", return_value=RunReturn())
+def test_chroot_context(mocked_run, mocked_check_call, tmp_path):
+
+    with Chroot(str(tmp_path)) as chroot:
```

Comment:
TIL. I did the same thing in https://github.com/osbuild/osbuild/blob/a2fed3e0d8e4d874c9345a89cd8c2e1bb2ef4da9/stages/test/test_dracut.py#L23 :innocent:

Reply:
It's fine :) It's super nitpick and the docs are not super clear here, it maybe that `tmp_path.as_posix()` is actually more correct but as long as we are not supporting windows it does not really matter afaict :)

Reply:
> TIL. I did the same thing in
> 
> https://github.com/osbuild/osbuild/blob/a2fed3e0d8e4d874c9345a89cd8c2e1bb2ef4da9/stages/test/test_dracut.py#L23
> 
> 😇

That's where I got it from of course :D 

But the path isn't even important here.  It could have just been `""`.

Reply:
I dropped the `tmp_path` and call it with `""` with a note.